ext/fiddle/fiddle.h

```diff
@@ -190,7 +189,7 @@
 #define ALIGN_INT32_T ALIGN_OF(int32_t)
 #define ALIGN_INT64_T ALIGN_OF(int64_t)
 
-#if RUBY_API_VERSION_MAJOR >= 3
+#ifdef HAVE_RUBY_MEMORY_VIEW_H
 # define FIDDLE_MEMORY_VIEW
```

Comment:
Can we remove `FIDDLE_MEMORY_VIEW` and use `HAVE_RUBY_MEMORY_VIEW_H` directly?

Reply:
Yes, unless you want to add other conditions.

Reply:
Could you also do it in this pull request?